packages/web-components/src/global/color-mode.css

```diff
@@ -421,6 +421,29 @@ ic-theme,
   --ic-popover-menu-border: var(--ic-color-border-neutral-white);
   --ic-popover-menu-item-lighten: var(--ic-brand-lighten-40);
 
+  /* Search bar */
+  --ic-search-bar-background: var(--ic-color-background-primary);
+  --ic-search-bar-placeholder-text: var(
+    --ic-atoms-input-text-field-content-placeholder
+  );
+  --ic-search-bar-filled-text: var(--ic-atoms-input-text-field-content);
+  --ic-search-bar-disabled-text: var(
+    --ic-atoms-input-text-field-content-disabled
+  );
+  --ic-search-bar-filled-icon: var(--ic-atoms-input-search-button);
+  --ic-search-bar-icon-disabled: var(--ic-state-layer-darken-20);
```

Comment:
just querying this one. my understanding is that the darken\lighten tokens should be used in addition to a base color, not just on their own. eg i would expect darken-20 to be something used on hover\press & not for a disabled color

Reply:
Not too sure, I assumed because the search bar now has a background colour for light and dark, it was using that as a base colour and then lightening/darkening that